torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1252,15 +1252,49 @@ Error PyTorchModelLoader::loadGlowFusedLinear(const torch::jit::Node *ptNode) {
   return addValueMapping(outputs[0], output);
 }
 
+Expected<NodeValue> PyTorchModelLoader::loadNodeValueOrBroadcastedConstant(
+    const torch::jit::Value *value, llvm::ArrayRef<size_t> dims) {
+  if (hasGlowNodeValueForValue(value)) {
+    return getGlowNodeValueForValue(value);
+  } else {
+    GlowIValue *ival;
+    ASSIGN_VALUE_OR_RETURN_ERR(ival, getGlowIValueForValue(value));
+    float constVal;
+    if (ival->isInt()) {
+      ASSIGN_VALUE_OR_RETURN_ERR(constVal,
+                                 static_cast_expected<float>(ival->toInt()));
+    } else {
+      ASSIGN_VALUE_OR_RETURN_ERR(constVal,
+                                 static_cast_expected<float>(ival->toDouble()));
+    }
+    glow::Tensor t(glow::ElemKind::FloatTy, dims);
+    t.init(glow::Tensor::InitKind::Broadcast, constVal,
+           F_.getParent()->getPRNG());
```

Comment:
What does `getPRNG` do?

Reply:
gets a PRNG 😛 
init takes a PRNG because if the InitKind is Xavier it would need it for random filling.

Reply:
Yeah, just to make sure that it won't actually random fill the tensor, right?

Reply:
That's right, it will not.